app/screens/participants_list/participants_list.tsx

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React, {PureComponent} from 'react';
+import {View} from 'react-native';
+import {UserProfile} from '@mm-redux/types/users';
+import type {Theme} from '@mm-redux/types/preferences';
+import SlideUpPanel from 'app/components/slide_up_panel';
+import {changeOpacity, makeStyleSheetFromTheme} from 'app/utils/theme';
+import FormattedText from 'app/components/formatted_text';
+import ParticipantRow from './participant_row';
+import {dismissModal} from 'app/actions/navigation';
+
+interface ParticipantsListProps {
+    userProfiles: UserProfile[];
+    teammateNameDisplay: string,
+    theme: Theme;
+}
+
+export default class ParticipantsList extends PureComponent<ParticipantsListProps> {
+    close = () => {
+        dismissModal();
+    };
+
+    renderHeader = () => {
+        const {theme} = this.props;
+        const style = getStyleSheet(theme);
+        return (
+            <View style={style.header}>
+                <FormattedText
+                    id='mobile.participants_header'
+                    defaultMessage={'THREAD PARTICIPANTS'}
+                    style={style.headerText}
+                />
+            </View>
+        );
+    }
+
+    renderParticipantRows = () => {
+        const {userProfiles, teammateNameDisplay, theme} = this.props;
+        return userProfiles.map((user: UserProfile) => (
```

Comment:
Why this is not a FlatlList component? What happens if there are 20+ participants? are we not going to show them all?

Reply:
I'll update to use a FlatList.

I'm not sure I understand the second part of your comment. What is the issue when there are 20+ participants? They are displayed correctly, the user just has to scroll. 
Do you mean to say performance will suffer? Looking at the docs for FlatList, its wrapper around VirtualizedList, which is definitely better than the current implementation.

Reply:
Slideup panel is using a scrollview already? If that is the case you can ignore this comment

Reply:
>Slideup panel is using a scrollview already?

Yep, it uses `Animated.ScrollView`